web-ng/.prettierrc.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
-  ...require('gts/.prettierrc.json'),
```

Comment:
This gives me error locally that cannot find gts/.prettierrc.json. Also why are we importing and adding properties both?